spec/services/curation_concerns/workflow/status_list_service_spec.rb

```diff
@@ -4,6 +4,7 @@
   describe "#each" do
     let(:user) { create(:user) }
     let(:service) { described_class.new(user) }
+    let(:sipity_entity) { instance_double(Sipity::Entity, workflow: workflow, workflow_state: workflow_state) }
```

Comment:
Why is this test using instance doubles when the others are using factories? If you can make this test use the same factories, that would bring extra consistency.